packages/form-js-editor/test/spec/core/Modeling.spec.js

```diff
@@ -21,12 +19,6 @@ describe('core/Modeling', function() {
 
   beforeEach(bootstrapFormEditor(schema));
 
-  beforeEach(async function() {
-    await waitFor(inject(function(formFieldRegistry) {
-      expect(formFieldRegistry.size).to.equal(11);
-    }));
-  });
-
   afterEach(function() {
     getFormEditor().destroy();
```

Comment:
Should also not be necessary, as the form editor created via our test helpers is automatically destroyed before a new one gets created.

Reply:
🧠